JustSaying/Messaging/IMessagePublisher.cs

```diff
@@ -9,5 +10,7 @@ public interface IMessagePublisher
         void Publish(Message message);
 #endif
         Task PublishAsync(Message message);
+
+        Task PublishAsync(Message message, CancellationToken cancellationToken);
```

Comment:
What version are we revving to next? Changing an interface is a breaking change, so this would make it 6?

Reply:
Yeah, this will need to be a major change (Just Saying 6, only a week or two after 5 was released 😬

Reply:
But this is only adding a new overload, so it's not a breaking change.

Reply:
Overloads are not optional though

Reply:
Ah I see the the chat comment. Yes this is on a public interface, so any implementers with be broke.